packages/web/src/instrumentation/documentLoadInstrumentation.js

```diff
@@ -7,17 +7,19 @@ import EpsagonUtils from '../utils';
 const api = require('@opentelemetry/api');
 
 class EpsagonDocumentLoadInstrumentation extends DocumentLoadInstrumentation {
-  constructor(parentSpan) {
+  constructor(parentSpan, isErrorCollectionDisabled) {
```

Comment:
I think it would be better to pass this as a part of the configurations rather than constructor parameter

Reply:
@nemoshlag What configuration are you referring to? The configuration in `web-tracer.init`?

Reply:
@schultyy Yes, I mean if I understand the feature correctly we want the end-user to be able to modify it as other configurations

Reply:
@nemoshlag Correct. By default, this flag is set to `false` so that everything behaves as usual. Only when the user specifies the flag like this: `epsagon.init({isTest: true, token: 'abcdef', errorDisabled: true})` errors won't be collected. 

Does that go into the right direction?

Reply:
Hi sorry missed your response. I think it's OK